docker-compose.yml

```diff
@@ -141,10 +141,10 @@ services:
       - ./test:/app/test
       - ./features:/app/features
 
-  react-native-cli:
+  react-native-cli-tool-maze-runner:
     build:
       context: .
-      dockerfile: dockerfiles/Dockerfile.react-native-cli
+      dockerfile: dockerfiles/Dockerfile.react-native-cli-tool
```

Comment:
what's with the new name?

Reply:
I'm anticipating needing to add a separate Dockerfile for the MazeRunner "end-to-end" (i.e. built and run-on-device) tests, so this is seeking to distinguish from that.